### PR TITLE
feat(matcssi_cms)

### PR DESCRIPTION
**Pull request by wesleyboar**
## Overview

Add MatCSSI project.

## Changes

- **added** `matcssi_cms`
- **added** instructions to README for a project with a `secrets.default.py`

### Remote

https://pprd.mise.tacc.utexas.edu/

### Local

Run MATCSSI with `make`. Set up CMS secrets, settings, container commands, et cetera. Load http://0.0.0.0:8000/.

<details>

0. `git clone git@github.com:TACC/Core-CMS-Custom.git`
1. `git checkout task/matcssi_cms`
2. ```
   cd custom-project-dir
   cp taccsite_cms/secrets.default.py taccsite_cms/secrets.py
    ```
3. Create a `settings_local.py` with content from [Core-CMS `settings_local.example.py`](https://github.com/TACC/Core-CMS/blob/main/taccsite_cms/settings_local.example.py).
4. `make build`
5. `make start`
6. Open new terminal.
7. `docker exec -it matcssi_cms /bin/bash`
8. `python manage.py migrate`
9. `python manage.py collectstatic`
10. `python manage.py createsuperuser`
11. Open http://0.0.0.0:8000/.
12. Populate as desired.

</details>

## UI

| remote | local |
| - | - |
| <img width="370" alt="macssi remote" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/fa9bf342-759e-402c-9763-a9aef8c48581"> | <img width="475" alt="matcssi local" src="https://github.com/TACC/Core-CMS-Custom/assets/62723358/a9bb8184-e734-490c-97aa-8dd778716c4a"> |

## Notes

To support running multiple CMS projects locally at once:

1. User must copy `secrets.default.py` to `secrets.py` or else docker network access would fail.
2. Docker containers are prefixed with `matcssi_` instead of `core_` or else container names would conflict.

### CODE REVIEW

File: matcssi_cms/src/taccsite_custom/TODO.md
Comment (taoteg) [inline matcssi_cms/src/taccsite_custom/TODO.md:1]: If this is done in this commit, maybe we can drop this file?

File: matcssi_cms/src/apps/custom_example/__init__.py
Comment (taoteg) [inline matcssi_cms/src/apps/custom_example/__init__.py:1]: Does the matcssi_cms need the example app? Or would that be better in the core cms demo/example site cms setup?

**Review by taoteg — APPROVED** [review]
LGTM. I left two comments, both more question than action and both up to you (if valid). Approving =).

File: matcssi_cms/src/taccsite_custom/TODO.md
Comment (wesleyboar) [reply matcssi_cms/src/taccsite_custom/TODO.md:1]: This is cruft—copy-pasta from APCD. I will delete this file.

File: matcssi_cms/src/apps/custom_example/__init__.py
Comment (wesleyboar) [reply matcssi_cms/src/apps/custom_example/__init__.py:1]: Nope, it does not need it. Yup, demo/example site CMS should have it. I will remove it from here.

Comment (wesleyboar) [reply matcssi_cms/src/apps/custom_example/__init__.py:1]: Removed. And I added #163 for example app. As you have bandwidth, let me know if that's what you were thinking.

**Pull request merged**